guides/v2.1/cloud/env/variables-global.md

```diff
@@ -0,0 +1,72 @@
+---
+group: cloud-guide
+title: Global variables
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+The following _global_ variables control actions in the build, deploy, and post-deploy stages of the `.magento.env.yaml` file. Because global variables impact every stage, you must set them in the `global` stage. Insert these variables in the `global` stage of the `.magento.env.yaml` file:
+
+```yaml
+stage:
+  global:
+    GLOBAL_VARIABLE_NAME: value
+```
+### `MIN_LOGGING_LEVEL`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Overrides the minimum logging level for all output streams without making changes to the code. This helps to improve troubleshooting problems with deployment. For example, if your deployment fails, you can use this variable to increase the logging granularity globally. See [Set up notifications—Log levels]({{ page.baseurl }}/cloud/env/setup-notifications.html#log-levels). The `min_level` value in Logging handlers overwrites this setting.
```

Comment:
"This helps to improve troubleshooting problems with deployment."  -- sounds a bit odd; makes it easier to troubleshoot .... or helps when troubleshooting?